Components/Alert/Alert.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./alertstyles.css" />
+    <title>Alert Component</title>
+  </head>
+  <body>
+    <h1>Alerts</h1>
+    <h2>Simple Alerts</h2>
+    <div class="alerts">
+      <div class="error-alert-background alert">
+        <svg
+          class="alert-image"
+          focusable="false"
+          viewBox="0 0 24 24"
+          aria-hidden="true"
```

Comment:
Maybe create an ``img.svg`` file and include it under ``<img />`` tag. 
It will be semantically more correct I guess & will reduce your codebase.